xgo.go

```diff
@@ -315,11 +315,17 @@ func compile(image string, config *ConfigFlags, flags *BuildFlags, folder string
 	re := regexp.MustCompile("([A-Z]):")
 	folder_w := filepath.ToSlash(re.ReplaceAllString(folder, "/$1"))
 	depsCache_w := filepath.ToSlash(re.ReplaceAllString(depsCache, "/$1"))
+	gocache := filepath.Join(depsCache, "gocache")
+	if err := os.MkdirAll(gocache, 0641); err != nil {
```

Comment:
is this `0641` intended?

Reply:
It was an attempt to shrink down from 644 suggested above: https://github.com/techknowlogick/xgo/pull/116#discussion_r629225549

Reply:
This is supposed to be a directory 0644 = rw-r--r-- which means you can't actually traverse the directory

This should be 0755 or 0750

Reply:
I'm gonna push up an update